app/javascript/articles/__tests__/Article.test.jsx

```diff
@@ -0,0 +1,150 @@
+import { h } from 'preact';
```

Comment:
The usual snapshot tests. In the future, with a newer version of Storybook, we can make use of [Storyshots](https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core) so that snapshot testing becomes part of building out your components automatically.